File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.py
class Solution(object):
    def findNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0
        for i in nums:
            if((i>9 and i<100)or(i>999 and i<10000)or(i>99999 and i<1000000)):
                 count += 1
        return count
        

File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for(int i:nums)if((i>9 && i<100)||(i>999 && i<10000)||(i>99999 && i<1000000)) count++;
        return count;
    }
}